data/unit_tests/make_mock_dataset.py

```diff
@@ -0,0 +1,90 @@
+import json
```

Comment:
I'm not sure I understand the rationale for this - if we have data already gathered from fr24 and other sources, would it not be better to use a subset of that?

Reply:
Oh is using a subset okay? I thought we didn't want to upload the dataset to the repo?